src/viam/resource/easy_resource.py

```diff
@@ -41,7 +88,7 @@ async def get_readings(self, **kwargs):
     See examples/easy_resource/main.py for extended usage.
     """
     SUBTYPE: Subtype
-    MODEL: Model
+    MODEL: Union[Model, str]
```

Comment:
I'm not sure how I feel about this -- I think this leaves a lot more chance for a user to provide a malformed/nonsense string

Reply:
reverted in 26c2b7e8

if this ends up causing type problems we can solve at leisure as separate project